models/openacademy_course.py

```diff
@@ -10,8 +11,13 @@ class Course(models.Model):
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
 
     _sql_constraints = [
-        ("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."),
-        ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")
+        ("description_and_title_different",
```

Comment:
This is an unrelated change. Please open a new PR to address only lints fixing (as I commented).

Reply:
This change comes from the rebasing with the 15.0 branch.